### ISSUE TITLE
"Error: Please install xflux in the PATH" when running fluxgui from Git clone

### ISSUE THREAD

**Issue by phoenixcomm** [xflux-gui/xflux-gui#40]
I have a fresh + updates of Linux Mint 18 (some things dont work... )

```
First I can not update or install your ppa as i get the 
```
This PPA does not support xenial
Cannot add PPA: ''This PPA does not support xenial''.
```
``` I have fixed the issue with the ppa :)
------
```
harrison@Whirlwind ~/Downloads/Software/Gits/xflux-gui $ PYTHONPATH=`pwd`/src:$PYTHONPATH ./fluxgui &
[1] 10945
harrison@Whirlwind ~/Downloads/Software/Gits/xflux-gui $ 
Error: Please install xflux in the PATH 
Critical error. Exiting.
Traceback (most recent call last):
File "./fluxgui", line 20, in <module>
    main()
File "/home/harrison/Downloads/Software/Gits/xflux-gui/src/fluxgui/fluxapp.py", line 213, in main
    app = FluxGUI()
File "/home/harrison/Downloads/Software/Gits/xflux-gui/src/fluxgui/fluxapp.py", line 28, in __init__
    self.exit(1)
File "/home/harrison/Downloads/Software/Gits/xflux-gui/src/fluxgui/fluxapp.py", line 46, in exit
    gtk.main_quit()
RuntimeError: called outside of a mainloop
```
----------------

**Comment by ntc2** [comment]
For installing directly from the Git repo, try running this in your `xflux-gui` Git clone:
```shell
 PATH=`pwd`:$PATH PYTHONPATH=`pwd`/src:$PYTHONPATH ./fluxgui
```
Does that fix your problem?

I'll update the README with the additional `PATH` setting ...

**Comment by ntc2** [comment]
Also, you'll need to run
```shell
python download-xflux.py
```
first.

**Comment by ntc2** [comment]
Going to assume this problem is resolved, since we never heard back.

**Comment by prestonvanloon** [comment]
Can you add it to the manual install as well? I get the same issue when following the manual install instructions.

**Comment by ntc2** [comment]
I don't understand: the `python download-xflux.py` is already listed in README in the manual install section. Or are you talking about something else? What does "it" refer to?

**Comment by prestonvanloon** [comment]
"it" = the fix.

To reproduce:
- On a linux, run the manual install instructions only. 
- Get this exciting error: 
```
Error: Please install xflux in the PATH 

Critical error. Exiting.
Traceback (most recent call last):
  File "/usr/local/bin/fluxgui", line 20, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/fluxgui/fluxapp.py", line 213, in main
    app = FluxGUI()
  File "/usr/local/lib/python2.7/dist-packages/fluxgui/fluxapp.py", line 28, in __init__
    self.exit(1)
  File "/usr/local/lib/python2.7/dist-packages/fluxgui/fluxapp.py", line 46, in exit
    gtk.main_quit()
RuntimeError: called outside of a mainloop
```
- Open this bug to find out you need to do this:
```
 PATH=`pwd`:$PATH PYTHONPATH=`pwd`/src:$PYTHONPATH ./fluxgui
```
instead of this
```
# Run flux
fluxgui
```

And that only works if you launch the fluxgui from the cloned repository. 
Running the globally installed `fluxgui` does not work with the manual install instructions. 
Purhaps `python download-xflux.py` is not actually installing `xflux`.
Is it supposed to be run as `sudo`? No error is thrown. 
```
➜  xflux-gui git:(master) ✗ python download-xflux.py
Downloading 64-bit xflux ...
--2017-02-21 14:40:51--  https://justgetflux.com/linux/xflux64.tgz
Resolving justgetflux.com (justgetflux.com)... 216.176.200.22
Connecting to justgetflux.com (justgetflux.com)|216.176.200.22|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 339845 (332K) [application/octet-stream]
Saving to: ‘/tmp/xflux.tgz’

100%[=======================================================================================================>] 339,845     1.63MB/s   in 0.2s   

2017-02-21 14:40:52 (1.63 MB/s) - ‘/tmp/xflux.tgz’ saved [339845/339845]

xflux
➜  xflux-gui git:(master) ✗ xflux
zsh: permission denied: xflux
```

**Comment by ntc2** [comment]
Thanks for explaining!

The ```PATH=`pwd`:$PATH``` stuff is only relevant to running `fluxgui` directly out of the Git repo. If you manually install with `python setup.py install [--user]`, either locally (in your home dir) or globally, then `fluxgui` and `xflux` should be on your `PATH`, assuming that `~/.local/bin` is on your `PATH`. I just updated the instructions to mention that `~/.local/bin` needs to be on your `PATH` when manually installing locally.

**HOWEVER**, there was a bug in the global manual install that I just fixed: the `sudo python setup.py` was installing `xflux` in `/usr/local/bin` with the wrong permissions, because of a mistake in the `setup.py` file. I do the local install with `--user`, so I had not run into this issue.

The `download-xflux.py` is only supposed to download `xflux`, not install it. Your final example where you tried to run `xflux` and got "permission denied" was probably due to the bug I just fixed: the were attempting to run the globally installed `xflux` with bad permissions. Sorry about the confusion.
